server/command_create.go

```diff
@@ -100,6 +100,13 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 		return nil, true, fmt.Errorf("installation name %s is invalid: only letters, numbers, and hyphens are permitted", install.Name)
 	}
 
+	if exists, err := p.installationWithNameExists(install.Name); err != nil || exists {
```

Comment:
nit: style with `if` is a bit different than the rest of the code, but 0/5 on needing to change it.

Reply:
Yep, sorry, old habits die hard, this is my personal style bleeding through. I'll correct this to adhere to project style.

It should be noted that the behavior is subtly different if these two variables are declared on their own line, as they then remain in scope / memory longer. I don't think it matters here but I do feel it's worth pointing out that the difference between this statement and project style isn't always _just_ style (although it essentially is just style in this case).